ui/components/ticket-to-space/SweepstakesWinners.tsx

```diff
@@ -68,40 +68,41 @@ function WinnerSkeleton() {
 export function SweepstakesWinners({ ttsContract, supply }: any) {
   const [winners, setWinners] = useState<Winner[]>([])
 
-  async function getWinners() {
```

Comment:
Do we need any of this logic any more? Could we just get rid of this entirely?

Reply:
I can archive it but I think we should keep the code.